### PR TITLE
[HYDRA-682] Refactor spark plugins into spark directory

### PR DESCRIPTION
**Pull request by knazir**
Issue: https://issues.cask.co/browse/HYDRA-682
Build: http://builds.cask.co/browse/HYP-BAD67-3

### CODE REVIEW

File: spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/TwitterStreamingSource.java [hunk 1]
```diff
@@ -31,8 +31,6 @@
 import org.apache.spark.streaming.api.java.JavaDStream;
 import org.apache.spark.streaming.api.java.JavaStreamingContext;
 import org.apache.spark.streaming.twitter.TwitterUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
```
Comment (knazir) [inline spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/TwitterStreamingSource.java:35]: These were unused imports that IntelliJ removed automatically while refactoring.

**Comment by sreevatsanraman** [comment]
LGTM

**Comment by albertshau** [comment]
Please check that one of these plugins works after the change. I believe you need to change the exportcontents in the pom.

**Comment by knazir** [comment]
Will try it out now @albertshau

**Comment by albertshau** [comment]
lgtm assuming you've deployed the jar and verified the plugin is available.

**Comment by knazir** [comment]
Deploying the jar still works. Will merge once guild is breen.

**Comment by knazir** [comment]
Deploying worked, guild is breen: http://builds.cask.co/browse/HYP-BAD67 going to merge.

**Pull request merged**